pkl-core/src/main/java/org/pkl/core/SecurityManagers.java

```diff
@@ -294,11 +305,11 @@ public StandardBuilder setRootDir(@Nullable Path rootDir) {
 
     @Override
     public SecurityManager build() {
-      if (allowedResources.isEmpty() && allowedModules.isEmpty()) {
-        throw new IllegalStateException("No security manager set.");
+      if (allowedModules.isEmpty()) {
+        throw new IllegalStateException("allowedModules cannot be empty");
```

Comment:
It would also error if `pkl:` is not part of the allowed modules.

Not actually sure that this check is that meaningful. It's not good enough as it stands, and the error from the evaluator is already pretty clear.